en/docs/develop-components/request-for-workflow-approval.md

```diff
@@ -0,0 +1,27 @@
+# Request for Workflow Approval
+
+In Choreo, if an administrator [configures a workflow to require approval](../administer/configure-approvals-for-choreo-workflows.md), you must submit a request to obtain approval to perform the task.
+
+When an approval request is made, Choreo creates a request and notifies all authorized assignees via email about the [review request](../administer/review-workflow-approval-requests.md). When an authorized assignee approves or rejects the request, you will receive an email notification regarding the decision.
+
+The approach to request approval can vary depending on the workflow. 
+
+## Request approval for environment promotion
+
+**Prerequisites**:
+
+- Ensure you have a component created, built, and deployed to the development environment.
+- Ensure that an approval is configured for the environment promotion workflow.
```

Comment:
`environment promotion` is enabled only when you are deploying to a `critical` environment.